spec/models/piece_spec.rb

```diff
@@ -8,4 +8,36 @@
         .to eq('white-queen.png')
     end
   end
+  describe "#move_to!" do
+    let(:king) { create(:king, x_position: 1, y_position: 1) }
```

Comment:
so just as an FYI, this is currently creating a piece on top of another existing piece (a pawn from when the board was initialized with all 32 original pieces). due to the way you set up your tests, this will pass because you're just referencing the king. BUT once we implement validations/checks to ensure that pieces are not occupying the same square in the same game, this test will break. 

two possible solutions: 1) manipulate the 32 pieces from their starting positions to get the tests you want  2) delete all the pieces from a single game and then use your piece factories to create new pieces in the positions that you want

you may not have to do this now (since we currently  don't check for stacked pieces), but somewhere down the line we will have to, so might be worth it to set it up correctly now?